### DISCUSSION TITLE
报错配置文件错误如下，求大佬解决一下

### DISCUSSION THREAD

**Discussion by dongfangmu** [zdz/ServerStatus-Rust#174] [Q&A]
✨ stat_server v1.7.0 (def692b, 2023-02-10 16:50:24 UTC, rustc 1.67.0, x86_64-unknown-linux-musl)
✨ run in cloud mode, load config from env
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { inner: Error { inner: TomlError { message: "expected `.`, `=`", original: Some("[https://raw.githubusercontent.com/zmyeir/ServerStatus-Rust/master/config.toml"](https://raw.githubusercontent.com/zmyeir/ServerStatus-Rust/master/config.toml%22);), keys: [], span: Some(5..6) } } }', server/src/config.rs:173:51
stack backtrace:
0:     0x7eff30b86710 - <unknown>
1:     0x7eff3094a3be - <unknown>
2:     0x7eff30b61934 - <unknown>
3:     0x7eff30b87c5f - <unknown>
4:     0x7eff30b8785f - <unknown>
5:     0x7eff30b888d2 - <unknown>
6:     0x7eff30b883b4 - <unknown>
7:     0x7eff30b8831c - <unknown>
8:     0x7eff30b882f1 - <unknown>
9:     0x7eff3083db32 - <unknown>
10:     0x7eff3083df62 - <unknown>
11:     0x7eff308d39b6 - <unknown>
12:     0x7eff308fd885 - <unknown>
13:     0x7eff308f9791 - <unknown>
14:     0x7eff3086658e - <unknown>
15:     0x7eff3086a26b - <unknown>
16:     0x7eff30907a9f - <unknown>

**Comment by dongfangmu** [comment]
使用的railway部署

**Comment by zdz** [comment]
先试试在最新的 config.toml 上面改，实在不行，就把敏感信息去掉，贴一下配置

**Reply by dongfangmu** [reply]
老哥，就用的你文件里那个配置文件，一点都没改，搭建就报错

**Reply by dongfangmu** [reply]
原文件搭建的报错，是需要一开始就改啥吗？加了账号密码也报错

**Reply by zdz** [reply]
默认不用改，railway 貌似要粘贴第二次才成功，对对文档

**Reply by dongfangmu** [reply]
确实是，粘贴了好几次，换文件也不行，老哥你看看这个文件

**Reply by dongfangmu** [reply]
https://raw.githubusercontent.com/dongfangmu/ServerStatus-Rust/master/config.toml

**Reply by zdz** [reply]
<img width="808" alt="image" src="https://user-images.githubusercontent.com/152173/228834048-3428475f-7c4c-46a1-8e51-eda28e2dcc6a.png">

我用你的配置没问题

**Reply by dongfangmu** [reply]
✨ stat_server v1.7.0 (def692b, 2023-02-10 16:50:24 UTC, rustc 1.67.0, x86_64-unknown-linux-musl)
✨ run in cloud mode, load config from env
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { inner: Error { inner: TomlError { message: "expected `.`, `=`", original: Some("[https://raw.githubusercontent.com/dongfangmu/ServerStatus-Rust/master/config.toml"](https://raw.githubusercontent.com/dongfangmu/ServerStatus-Rust/master/config.toml%22);), keys: [], span: Some(5..6) } } }', server/src/config.rs:173:51
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Reply by dongfangmu** [reply]
我用是这样

**Reply by zdz** [reply]
是把配置文件的内容粘贴到 railway，不是粘贴配置文件的链接。。。

**Reply by dongfangmu** [reply]
答疑解惑了老哥！！！！

**Reply by dongfangmu** [reply]
困扰好几天，非常感谢作者大大的回复！
